apps/src/tutorialExplorer/tutorialSet.jsx

```diff
@@ -14,76 +14,83 @@ const TutorialSet = React.createClass({
     specificLocale: React.PropTypes.bool
   },
 
-  /**
-   * Returns true if we should show this item based on current filter settings.
-   * It goes through all active filter categories.  If no filters are set for
-   * a filter group, then that item will default to showing, so long as no other
-   * filter group prevents it from showing.
-   * But if we do have a filter set for a filter group, and the tutorial is tagged
-   * for that filter group, then at least one of the active filters must match a tag.
-   * e.g. If the user chooses two platforms, then at least one of the platforms
-   * must match a platform tag on the tutorial.
-   * A similar check for language is done first.
-   * In the case that this.props.specificLocale is true, we do something slightly
-   * different.  We don't show tutorials that don't have any language tags, and we
-   * reject tutorials that don't have the current locale explicitly listed.  This
-   * allows us to return a set of tutorials that have explicit support for the
-   * current locale.
-   *
-   * @param {object} tutorial - Single tutorial, containing a variety of
-   *   strings, each of which is a list of tags separated by commas, no spaces.
-   */
-  filterFn(tutorial) {
+  statics: {
+    /**
+     * Filters a given array of tutorials by the given filter props.
+     *
+     * It goes through all active filter categories.  If no filters are set for
+     * a filter group, then that item will default to showing, so long as no other
+     * filter group prevents it from showing.
+     * But if we do have a filter set for a filter group, and the tutorial is tagged
+     * for that filter group, then at least one of the active filters must match a tag.
+     * e.g. If the user chooses two platforms, then at least one of the platforms
+     * must match a platform tag on the tutorial.
+     * A similar check for language is done first.
+     * In the case that filterProps.specificLocale is true, we do something slightly
+     * different.  We don't show tutorials that don't have any language tags, and we
+     * reject tutorials that don't have the current locale explicitly listed.  This
+     * allows us to return a set of tutorials that have explicit support for the
+     * current locale.
+     *
+     * @param {Array} tutorials - Array of tutorials.  Each contains a variety of
+     *   strings, each of which is a list of tags separated by commas, no spaces.
+     * @param {object} filterProps - Object containing filter properties.  Each is
```

Comment:
Might be worth also listing out props we expect in filterProps, i.e
```
@param {string} filterProps.locale
```
alternatively, you could begin your function by extracting those items
```
const { locale, specificLocale, filters } = filterProps;